extensions/amp-list/amp-list.md

```diff
@@ -70,15 +70,12 @@ In the following example, we retrieve JSON data that contains URLs and titles, a
   width="auto"
   height="100"
   layout="fixed-height"
-  src="{{server_for_email}}/static/inline-examples/data/amp-list-urls.json"
->
-  <template type="amp-mustache"
-    >{% raw %}
+  src="{{server_for_email}}/static/inline-examples/data/amp-list-urls.json">
+  <template type="amp-mustache">{% raw %}
```

Comment:
There needs to be a space between the closing `>` of the `template` and the `{% raw %}` block. Otherwise, prettier gets confused and moves the angular brace to the next line.

For example, this works:

```html
<amp-list
  width="auto"
  height="100"
  layout="fixed-height"
  src="{{server_for_email}}/static/inline-examples/data/amp-list-urls.json"
>
  <template type="amp-mustache">
    {% raw %}
    <div class="url-entry">
      <a href="{{url}}">{{title}}</a>
    </div>
    {% endraw %}
  </template>
</amp-list>
```